### ISSUE TITLE
Shortcut C+Shift+Up/Down for subscript/superscript broken

### ISSUE THREAD

**Issue by itsrainingdata** [SublimeText/LaTeXTools#1160]
In math environments such as align/align*, the old shortcut `C+Shift+Up/Down` to insert `^{}` / `_{}` seems to be broken. In inline math-mode with `$...$`, it still works fine.

To reproduce: Try typing x^{2} w/ shortcut in an align environment.

```
\begin{align}
x % shortcut C+Shift+Up/Down doesn't work here
\end{align}
```

**Comment by r-stein** [comment]
Seems that I have missed that when porting to the new syntax definition (also the keybinding only seem to be working correctly on OSX).
Add this to your keymap until I have fixed it:

``` js
	{ "keys": ["shift+super+down"], "command": "insert_snippet", "args": {"contents": "_{$1}$0"}, 
	"context":  
		[
			{"key": "selector", "operator": "equal", "operand": "text.tex.latex meta.environment.math"},
			{ "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
			{ "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true }
		]
	},

	{ "keys": ["shift+super+up"], "command": "insert_snippet", "args": {"contents": "^{$1}$0"}, 
	"context":  
		[
			{"key": "selector", "operator": "equal", "operand": "text.tex.latex meta.environment.math"},
			{ "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
			{ "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true }
		]
	},
```

**Comment by itsrainingdata** [comment]
Adding the suggested bindings works perfectly. Thanks!

**Comment by itsrainingdata** [comment]
I've noticed that these bindings keep getting reset back, although I am not sure what the cause is. I tried restarting Sublime and they persist beyond that, but every few weeks they seem to disappear.

**Comment by r-stein** [comment]
Add it to your Keybindings - User file not inside the LaTeXTools file, which will be revert with new updates.

**Comment by ig0774** [comment]
This should be fixed in v3.15.1.
